skymp5-server/cpp/server_guest_lib/PapyrusActor.cpp

```diff
@@ -178,3 +178,36 @@ VarValue PapyrusActor::SetDontMove(VarValue self,
   }
   return VarValue::None();
 }
+
+VarValue PapyrusActor::WornHasKeyword(VarValue self,
+                                      const std::vector<VarValue>& arguments)
+{
+  if (auto actor = GetFormPtr<MpActor>(self)) {
+    if (arguments.size() < 1) {
+      throw std::runtime_error(
+        "Actor.WornHasKeyword requires at least one argument");
+    }
+
+    const auto& keywordRec = GetRecordPtr(arguments[0]);
+    if (!keywordRec.rec) {
+      spdlog::error("Actor.WornHasKeyword - invalid keyword form");
+      return VarValue(false);
+    }
+
+    std::vector<Inventory::Entry> entries = actor->GetEquipment().inv.entries;
+    for (size_t i = 0; i < entries.size(); i++) {
+      const auto& keywordIds =
+        worldState->GetEspm()
+          .GetBrowser()
+          .LookupById(entries[i].baseId)
+          .rec->GetKeywordIds(worldState->GetEspmCache());
+      for (auto rawId : keywordIds) {
+        if (rawId == keywordRec.rec->GetId()) {
```

Comment:
both should be converted from *raw* to *global* format. 

Search for ToGlobalId for examples

PLease find a way to do this. then pass this PR to testing by SupAidmi